Refactor time in Lesson class into TimeInterval

itssisi opened the pull request:
Currently, the start time and end time of lesson is handled directly by the Lesson class.

For better OOP, the time class is refactor into TimeInterval class.

There is a bug on the initial display of the TA list. Any help is appreciated:)

Closes #53.

itssisi committed:
Refactor time in Lesson class into TimeInterval

Currently, the start time and end time of lesson is handled directly by the Lesson class.

For better OOP, the time class is refactor into TimeInterval class.

PohSayKeong commented:
Hi , please can consider adding a hashcode function for the `TimeInterval` class, I think that's causing the issue.

itssisi committed:
Fix bugs on TimeInterval

There is a bug when refactoring time interval in lesson, causing the initial addressbook to be empty when launched.

This is fixed by implementing hashcode inside the TimeInterval class

PohSayKeong reviewed (approved):
LGTM
